src/runtime/hexagon/hexagon_device_api.cc

```diff
@@ -233,6 +233,19 @@ TVM_REGISTER_GLOBAL("device_api.hexagon.dma_wait").set_body([](TVMArgs args, TVM
   *rv = static_cast<int32_t>(0);
 });
 
+TVM_REGISTER_GLOBAL("device_api.hexagon.dma_start_group")
+    .set_body([](TVMArgs args, TVMRetValue* rv) {
```

Comment:
Nit: When the input/output types are fixed, the `.set_body_typed()` method can be used to avoid needing manual argument wrangling.

```c++
.set_body_typed([](int queue_id) -> int32_t {
      return HexagonDeviceAPI::Global()->UserDMA()->StartGroup(queue_id);
    });
```